_template/api/proto/buf.yaml

```diff
@@ -6,11 +6,12 @@ deps:
   - buf.build/envoyproxy/protoc-gen-validate
   - buf.build/googleapis/googleapis
 {{- if .Extensions.grpc.grpcGateway }}
-  - buf.build/grpc-ecosystem/grpc-gateway
+  - buf.build/gnostic/gnostic
 {{- end }}
 lint:
   use:
     - DEFAULT
   except:
     - RPC_RESPONSE_STANDARD_NAME
     - RPC_REQUEST_RESPONSE_UNIQUE
+    - IMPORT_NO_WEAK # required by gnostic for now. this way gen-go will not import _ "" broken packages
```

Comment:
As I understood it, it is only used when gRPC Gateway is in use, so could you please only import it when gRPC Gateway was enabled:

```yaml
{{- if .Extensions.grpc.grpcGateway }}
    - IMPORT_NO_WEAK # required by gnostic for now. this way gen-go will not import _ "" broken packages
{{- end }}
```

Reply:
You are right!